pkg/service/backup/service.go

```diff
@@ -83,6 +83,20 @@ func (s *Service) Runner() Runner {
 	return Runner{service: s}
 }
 
+// GetRetentionFrom returns a function that returns the retention policy for a
+// given task ID, with a default of 30 days retention if no policy exists.
+func GetRetentionFrom(retentionMap map[uuid.UUID]Retention) func(uuid.UUID) Retention {
```

Comment:
Should all of this be exported, I mean Retention and all related structs and functions?